security/pkg/nodeagent/caclient/providers/google/client.go

```diff
@@ -110,3 +122,14 @@ func (cl *googleCAClient) getTLSDialOption() (grpc.DialOption, error) {
 	creds := credentials.NewClientTLSFromCert(pool, "")
 	return grpc.WithTransportCredentials(creds), nil
 }
+
+func parseZone(clusterURL string) string {
+	// input: https://container.googleapis.com/v1/projects/quanlinpilot1/locations/us-central1-c/clusters/mpibetacluster2
+	// output: quanlinpilot1 us-central1-c
```

Comment:
Is this output comment correct? IIUC, FindStringSubmatch() returns ["https://container.googleapis.com/v1/projects/quanlinpilot1/locations/us-central1-c/clusters/mpibetacluster2", "quanlinpilot1", "us-central1-c"]

Reply:
Anonymize `quanlinpilot1` to some general name?